src/functions/template-tags/html.php

```diff
@@ -132,3 +132,83 @@ function tribe_disabled( $disabled, $echo = true ) {
 		}
 	}
 }
+
+function tribe_format_field_dependency( $deps ) {
+    // Sanity check.
+    if ( empty( $deps ) ) {
+        return '';
+	}
+
+	// Let's be case-insensitive!
+	if ( isset( $deps['ID'] ) ) {
+		$deps['id'] = $deps['ID'];
+		unset( $deps['ID'] );
+	}
+
+    // No ID to hook to? Bail.
+    if ( empty( $deps['id'] ) ) {
+        return;
+    }
+
+    $dependency = '';
+
+    $accepted = [
+        'id',
+        'is',
+        'is-not',
+        'is-empty',
+        'is-not-empty',
+        'is-numeric',
+        'is-not-numeric',
+        'is-checked',
+        'is-not-checked',
+    ];
+
+    foreach ( $deps as $attr => $value ) {
+        // Attributes are always lower case.
+        $attr = strtolower( $attr );
+
+        // Valid dependency check.
+        // Note this means the dependency has to begin with "is", except for "id"!
+        if ( ! in_array( $attr, $accepted ) ) {
+            continue;
+        }
+
+        // Handle the ID component.
+        if ( 'id' === $attr ) {
+			// "Missing" quotes is intentional - they are added automagically and we want to avoid doubles.
+            $dependency .= " data-depends={$value}";
+            continue;
+        }
+
+        // Handle boolean values.
+        if ( is_bool( $value ) ) {
```

Comment:
Would this be covering values that are still considered booleans, but are not booleans, e.g. `0` or `1`?

Reply:
No - since we could be checking that a value is (int|string) 1  or (int|string) 0 I didn't want to convert.
ie "Show this section if the value of #id is exactly 1, but not if it's 0 or 2"?

Reply:
Ok, thanks for the clarification.